sw/device/silicon_creator/manuf/base/ft_personalize.c

```diff
@@ -607,14 +601,22 @@ bool test_main(void) {
   CHECK_STATUS_OK(personalize_extension_pre_cert_endorse(&pre_endorse));
 
   CHECK_STATUS_OK(personalize_endorse_certificates(&uj));
+  CHECK_STATUS_OK(log_hash_of_all_certs(&uj));
 
   personalize_extension_post_endorse_t post_endorse = {
       .uj = &uj,
       .perso_blob_from_host = &perso_blob_from_host,
       .cert_flash_layout = cert_flash_layout};
   CHECK_STATUS_OK(personalize_extension_post_cert_endorse(&post_endorse));
 
-  CHECK_STATUS_OK(log_hash_of_all_certs(&uj));
+  // Log the final hash of all certificates to the host and console.
+  serdes_sha256_hash_t hash;
+  hmac_sha256_process();
+  hmac_sha256_final((hmac_digest_t *)&hash);
+  CHECK_STATUS_OK(send_final_hash(&uj, &hash));
+  LOG_INFO("SHA256 hash of all certificates: %08x%08x%08x%08x%08x%08x%08x%08x",
+           hash.data[7], hash.data[6], hash.data[5], hash.data[4], hash.data[3],
+           hash.data[2], hash.data[1], hash.data[0]);
```

Comment:
This hash includes more than certs  (potentially) doesn't it?

```suggestion
  // Log the final hash of all assets written to flash (certificates and custom extension data)
  // that have been exported to the host. This is used to compare what was written to flash
  // vs. what was sent to the host.
  serdes_sha256_hash_t hash;
  hmac_sha256_process();
  hmac_sha256_final((hmac_digest_t *)&hash);
  CHECK_STATUS_OK(send_final_hash(&uj, &hash));
  LOG_INFO("SHA256 hash of all flash assets: %08x%08x%08x%08x%08x%08x%08x%08x",
           hash.data[7], hash.data[6], hash.data[5], hash.data[4], hash.data[3],
           hash.data[2], hash.data[1], hash.data[0]);
```

Reply:
done